consensus/state.go

```diff
@@ -538,6 +538,7 @@ func (cs *State) updateRoundStep(round int32, step cstypes.RoundStepType) {
 		}
 		if cs.Step != step {
 			cs.metrics.MarkStep(cs.Step)
+			schema.WriteRoundState(cs.traceClient, cs.Height, round, uint8(step))
```

Comment:
I guess my previous question applies here: Isn't `updateRoundStep` always followed by `newStep`? Then we already trace changes in `newStep` which means we will have duplicates?

Reply:
This is not always the case. For instance I notice the NewRound steo was never triggered and also the PrecommitWait step

Reply:
Oh and I removed the tracing from `newStep`